samples/boards/96b_argonkey/README.rst

```diff
@@ -0,0 +1,75 @@
+.. _ArgonKey:
+
+ArgonKey sensor board
+#####################
+
+Overview
+********
+This sample provides an example of how to read sensor data
+from the ArgonKey board. The result is displayed on the console.
+It makes use of both the trigger and poll methods.
+
+Requirements
+************
+
+This sample just requires the ArgonKey board.
+The user may select or unselect the sensors from
+:file:`samples/boards/argonkey/prj.conf`.
+
+Please note that all sensor related code is conditionally compiled
+using the `#ifdef` directive, so this sample is supposed to always
+build correctly. Example:
+
+ .. code-block:: c
+
+    #ifdef CONFIG_HTS221
+      struct device *hum_dev = device_get_binding("HTS221");
+
+      if (!hum_dev) {
+        printk("Could not get pointer to %s sensor\n", "HTS221");
+        return;
+      }
+    #endif
+
+References
+**********
+
+- :ref:`96b_argonkey`
+
+Building and Running
+********************
+
+ .. code-block:: bash
+
+    $ cd samples/boards/argonkey
```

Comment:
Use zephyr-app-commands instead. See samples/hello_world/README.rst for an example

Reply:
not clear to me. Why not using bash as a code block here. These are really bash shell commands

Reply:
ok, almost done.